core/client/helpers.js

```diff
@@ -79,14 +80,27 @@ toggleUserProperty = (propertyName, value) => {
         }
     }
 
-    if (typeof value === 'boolean')
+    let propertyValue
+
+    if (typeof value === 'boolean') {
+        propertyValue = !!value
+
         Meteor.users.update(user._id, {
-            $set: { [`profile.${propertyName}`]: !!value },
+            $set: { [`profile.${propertyName}`]: propertyValue },
         })
-    else
+    } else {
+        propertyValue = !user.profile[propertyName]
+
         Meteor.users.update(user._id, {
-            $set: { [`profile.${propertyName}`]: !user.profile[propertyName] },
+            $set: { [`profile.${propertyName}`]: propertyValue },
         })
+    }
```

Comment:
(I prefer this chunk as it was before 😬 )

Reply:
I've updated it, what about now ?

Reply:
Yeah that's better but in general you should avoid modifying code that doesn't need to be modified